tests/manage/ocs_288.py

```diff
@@ -0,0 +1,292 @@
+"""
+A test for creating pvc with random sc
+"""
+import logging
+import os
+import random
+
+import pytest
+
+from ocs import defaults, constants
+from ocs import ocp
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from resources import pod
+from resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+TEMPLATE_DIR = "templates"
```

Comment:
do you still need this here?
default.py holds the same

Reply:
Removed not required now